boards/arm/disco_l475_iot1/disco_l475_iot1.dts

```diff
@@ -6,6 +6,7 @@
 
 /dts-v1/;
 #include <st/l4/stm32l475Xg.dtsi>
+#include <l4/stm32l475r(c-e-g)tx-pinctrl.dtsi>
```

Comment:
Would probably be good to prefix pinctrl files with `st/`. hal_stm32 script can generate files in `dts/st` folder.

Reply:
Indeed, I initially though having `dts/st` in hal_stm32 would seem strange, but it is probably better from board point of view and this would allow moving .dtsi from one repo to another...